cloud-storage/onedrive/tool/src/main/java/org/sakaiproject/onedrive/ThymeleafConfig.java

```diff
@@ -0,0 +1,64 @@
+package org.sakaiproject.onedrive;
+
+import org.sakaiproject.util.ResourceLoaderMessageSource;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.ViewResolver;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.spring4.SpringTemplateEngine;
+import org.thymeleaf.spring4.templateresolver.SpringResourceTemplateResolver;
+import org.thymeleaf.spring4.view.ThymeleafViewResolver;
+import org.thymeleaf.templatemode.TemplateMode;
+import org.thymeleaf.templateresolver.ITemplateResolver;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("org.sakaiproject.onedrive")
+public class ThymeleafConfig extends WebMvcConfigurerAdapter implements ApplicationContextAware {
```

Comment:
Lets rename this to WebConfig as it's essentially the web.xml

Reply:
Thymeleaf just happens to be the template engine in use what happens if you change that :)